.github/pr-labeler.yml

```
@@ -1,3 +1,3 @@
 master: ['hotfix/*', 'release/*', 'dev/*', 'dev*']
 external-dependency: ['combined/*', 'combined-dependabots/*']
-brightlayer-ui: ['feature/*', 'bug/*', 'fix/*', 'experiment/*']
+brightlayer-ui: ['feature/*', 'bug/*', 'fix/*', 'experiment/*', 'dev/*']
```

won't that target only `dev/xxxxx` instead of `dev`?

```suggestion
brightlayer-ui: ['feature/*', 'bug/*', 'fix/*', 'experiment/*', 'dev']
```